pkg/api/kubernetes.go

```diff
@@ -5,8 +5,11 @@ type Kubernetes struct {
 	EncryptionAtRest  EncryptionAtRest         `yaml:"encryptionAtRest"`
 	Networking        Networking               `yaml:"networking,omitempty"`
 	ControllerManager ControllerManager        `yaml:"controllerManager,omitempty"`
+	KubeScheduler     KubeScheduler            `yaml:"kubeScheduler,omitempty"`
+	KubeProxy         KubeProxy                `yaml:"kubeProxy,omitempty"`
+	KubeLet           KubeLet                  `yaml:"kubeLet,omitempty"`
```

Comment:
I think this should be `Kubelet           Kubelet                  `yaml:"kubelet,omitempty"`` given it isn't called `kube-let` in kebab-case.